src/components/NavAuthMenu.tsx

```diff
@@ -14,7 +14,7 @@ import {
 } from "@mui/material";
 import { useState } from "react";
 import { Link } from "react-router";
-import type { User } from "@hooks/useAuth";
+import type { ApiUser as User } from "@marcinfras/map-report-server/types";
```

Comment:
Wg mnie dość dziwna praktyka to jest, aby dawać prefix "Api" przed typami. Ale to też pewnie kwestia subiektywna, więc up to you, czy to zmieniać